invofi/apps/frontend/src/lib/migrations/002_securitization.sql

```
@@ -0,0 +1,177 @@
+-- ── Row Level Security ────────────────────────────────────────────────────────
+
+alter table fractionalization_records   enable row level security;
+alter table fractional_positions        enable row level security;
+alter table price_history               enable row level security;
+alter table dividend_distributions      enable row level security;
+
+-- Fractionalization records: world-readable, originator can insert/update
+create policy "Anyone can read fractionalization records"
+  on fractionalization_records for select using (true);
+
+create policy "Originator can create fractionalization"
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)